sys/arm64/conf/GENERIC-MORELLO-PURECAP

```diff
@@ -0,0 +1,34 @@
+#
+# GENERIC-MORELLO-PURECAP -- Generic kernel configuration file for CheriBSD/Morello pure capability ABI kernel
+#
+# For more information on this file, please read the config(5) manual page,
+# and/or the handbook section on Kernel Configuration Files:
+#
+#    https://www.FreeBSD.org/doc/en_US.ISO8859-1/books/handbook/kernelconfig-config.html
+#
+# The handbook is also available locally in /usr/share/doc/handbook
+# if you've installed the doc distribution, otherwise always see the
+# FreeBSD World Wide Web server (https://www.FreeBSD.org/) for the
+# latest information.
+#
+# An exhaustive list of options and more detailed explanations of the
+# device lines is also present in the ../../conf/NOTES and NOTES files.
+# If you are in doubt as to the purpose or necessity of a line, check first
+# in NOTES.
+#
+# $FreeBSD$
+
+include "GENERIC-MORELLO"
+
+ident		GENERIC-MORELLO-PURECAP
+
+machine 	arm64 aarch64c
+
+makeoptions	CHERI_SUBOBJECT_BOUNDS=subobject-safe
+
+options 	CHERI_PURECAP_KERNEL
+
+# Use FDT instead of ACPI discovery
+nodevice 	acpi
+options 	FDT_DTB_STATIC
+makeoptions 	FDT_DTS_FILE=morello-fvp.dts
```

Comment:
How does this work on QEMU?

Reply:
It does not, I have a local change with a QEMU DTS file instead, that is waiting to be pushed until I finish to debug it.

Reply:
Which then only works for a specific QEMU config, namely the default cheribuild one. :( We probably need to think about tackling ACPI soon...

Reply:
Sadly yes. I just don't want to get into it for the release. We will need to drop the QEMU flag from the generic one in cheribuild.

Reply:
Adding the QEMU config file in a separate commit as well.